pkg/channeld/data.go

```diff
@@ -174,6 +178,7 @@ func (ch *Channel) tickData(t ChannelTime) {
 			tmp := focp.Next()
 			ch.fanOutQueue.Remove(focp)
 			focp = tmp
+			foci--

```

Comment:
Unnecessary as ch.fanOutQueue.Len() is checked for each loop

Reply:
refactored this logic, removed this value.